contracts/address-manager/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "address-manager"
+version = "0.1.0"
+authors = ["Zeke Medley <zekemedley@gmail.com>"]
+edition = "2018"
+
+exclude = [
+  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
+  "contract.wasm",
+  "hash.txt",
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[package.metadata.scripts]
+optimize = """docker run --rm -v "$(pwd)":/code \
+  --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+  --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+  cosmwasm/rust-optimizer:0.12.4
+"""
+
+[dependencies]
+cosmwasm-std = { version = "1.0.0-beta4" }
+cosmwasm-storage = { version = "1.0.0-beta4" }
```

Comment:
You can leave off the `4` here and it will use the latest version. For example: `"1.0.0-beta"`.